docs/usage.rst

```diff
@@ -414,7 +414,7 @@ is not built into pyjwt.
     signing_key = jwks_client.get_signing_key_from_jwt(id_token)
 
     # now, decode_complete to get payload + header
-    data = jwt.decode_complete(
+    data = jwt.jwt_api.decode_complete(
```

Comment:
oh instead of this lets surface https://github.com/jpadilla/pyjwt/blob/master/jwt/api_jwt.py#L432 in the __init__ https://github.com/jpadilla/pyjwt/blob/master/jwt/__init__.py#L9